repose-aggregator/artifacts/docker/build.gradle

```diff
@@ -6,13 +6,9 @@ import groovy.json.JsonSlurper
 
 apply plugin: 'com.bmuschko.docker-remote-api'
 
-// This forces the usage of a more recent unix-socket-factory which makes Mac OS X happy
-// It also forces the usage of jackson dependencies which were not being brought in as expected
-// TODO: Remove unix-socket-factory after upgrading to gradle-docker-plugin 3.0.4
 configurations {
     dockerJava {
         resolutionStrategy {
-            force 'de.gesellix:unix-socket-factory:2016-04-06T22-21-19'
             force 'com.fasterxml.jackson.core:jackson-databind:2.6.4'
             force 'com.fasterxml.jackson.core:jackson-core:2.6.4'
             force 'com.fasterxml.jackson.core:jackson-annotations:2.6.4'
```

Comment:
Were these dependencies fixed by the upgrade? I would love not to have to keep this around.

If not, we should maybe keep the comment about what this is for around. If that comment isn't clear enough, our transitive dependency on Jackson was not resolving its dependencies correctly, so I forced them here to prevent version mismatch issues. That seems like an issue with either docker-java or jackson, so we may be able to remove our patch.

Reply:
Do you remember what the Jackson stuff was causing to fail?

Reply:
Not exactly, but without it, at least one of the Docker tasks would fail. So if you can remove this and run all of the tasks, we should be good.

It looks like the `docker-java` library uses Jackson to interact with the Docker API, so...

Reply:
Considering that `docker-java` is still on Jackson 2.6.4, and that they haven't added any new dependencies, I would guess this is still an issue:
https://github.com/docker-java/docker-java/blob/master/pom.xml#L93-L97

... but I could be wrong.

Reply:
I can successfully execute both of the following with it removed:

* `:repose-aggregator:artifacts:docker:buildUbuntuImage`
* `:repose-aggregator:artifacts:docker:buildCentosImage`

Reply:
Noice.